ArduPlane/quadplane.cpp

```diff
@@ -709,6 +711,12 @@ bool QuadPlane::setup(void)
     motors->set_update_rate(rc_speed);
     attitude_control->parameter_sanity_check();
 
+    // Try to convert mot PWM params, if still invalid force conversion
+    AP_Param::convert_old_parameters(&mot_pwm_conversion_table[0], ARRAY_SIZE(mot_pwm_conversion_table));
+    if ((motors->get_pwm_output_min() <= 0) || (motors->get_pwm_output_max() <= 0) || (motors->get_pwm_output_max() <= motors->get_pwm_output_min())) {
+        AP_Param::convert_old_parameters(&mot_pwm_conversion_table[0], ARRAY_SIZE(mot_pwm_conversion_table), 2);
```

Comment:
looks good, thanks! but we should add flag defines in AP_Param.h for these conversion flags

Reply:
done